subsys/usb/usb_c/Kconfig

```diff
@@ -38,6 +38,14 @@ config BUILD_OUTPUT_VIF
 	depends on DT_HAS_USB_C_CONNECTOR_ENABLED
 	help
 	  Generate XML file containing VIF policies during project build.
+choice
+	prompt "USB-C Connection State Machine"
+	default USBC_SNK_CSM
+
+config USBC_SNK_CSM
+	bool "Sink USBC Connection State Machine"
+
```

Comment:
Done

Reply:
Nit - why is sink capitalized?  Also, specify that this option configures the CSM for sink operation only.  Because this is a choice symbol I assume that you will also eventually add a DRP option.

What do you think about naming this option `USBC_CSM_SINK_ONLY`?

Reply:
"Sink" and "Source" are proper nouns in the Type-C spec.

Reply:
I changed the name